### PR TITLE
rustpython-stdlib crate

### PR DESCRIPTION
**Pull request by youknowone**
#3102

### CODE REVIEW

**Comment by youknowone** [comment]
done except for windows

**Comment by youknowone** [comment]
@DimitrisJim @coolreader18 there will be more things to do (e.g. corelib), but I want to finish this PR for current state and do further works later.

File: stdlib/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+#[cfg(target_os = "macos")]
+mod scproxy;
+#[cfg(not(target_arch = "wasm32"))]
+mod select;
+#[cfg(all(not(target_arch = "wasm32"), feature = "ssl"))]
+mod ssl;
+#[cfg(all(unix, not(target_os = "redox")))]
+mod termios;
+
+use rustpython_common as common;
```
Comment (youknowone) [inline stdlib/src/lib.rs:51]: is this intended to be used like `vm::common`?

**Review by coolreader18 — APPROVED** [review]

**Comment by coolreader18** [comment]
LGTM!

**Review by DimitrisJim — APPROVED** [review]

**Pull request merged**